### ISSUE TITLE
Y24-426 - As Legal, we would like to test that the Data Access Agreements actually used for Studies are synchronised with the Electronic Data Access Management (EDAM) system to avoid delays in granting access. 

### ISSUE THREAD

**Issue by harrietc52** [sanger/sequencescape#4453]
**User story**
As LeGo, we would like to test that the Data Access Agreements actually used for Studies are synchronised with the Electronic Data Access Management (EDAM) system to avoid delays in granting access.  

This story is acting as a Test Case

**Who are the primary contacts for this story**
@harrietc52 

**Who is the nominated tester for UAT**
Liz Huckle, Liz Cook (SSR)
Odion A (eDAM2 developer)
Laura B (LeGo, Data Access Officer)
Melanie Sharp (DAA)
Kalia Dede/ Mugdha K (HumGen/ GenGen) 
Quan Lin (Data submission team)
Francesca Beaton, Ali Hitchborn (CellGen)

**Acceptance criteria**
To be considered successful the solution must allow:

- [ ] 1. User(?) creates Study in SS UAT
- [ ] 2. eDAM confirms study is created in eDAM
- [ ] 3. eDAM confirms a DAA has been created
- [ ] 4. User(?) uploads DAA to SS Study page
- [ ] 5. User(?) can access DAA from SS
- [ ] 6. User(?) updates Study in SS
- [ ] 7. MLWH/ eDAM/ DAA/ EBI/ DRT are all updated 
- [ ] 8. Updated DAA is reuploaded to SS
- [ ] 9. User(?) can access updated DAA from SS
- [ ] 10. User(?) creates a second Study in SS UAT (using the CSV upload? - Non-MVP)
- [ ] 11. A second Study is created in SS UAT
- [ ] 12. Samples are uploaded to SS under Study 1
- [ ] 13. Samples are sequenced under Study 2
- [ ] 14. MLWH data for samples ref study 2 (not study 1) 
- [ ] 15. LeGo can easily access DAA for study 2 

**Dependencies**
This story is blocked by the following dependencies:
[Y24-419](https://github.com/sanger/sequencescape/issues/4443) - As LeGo, we would like additional fields to be added to the Study setup form in Sequencescape, so users don’t have to input data twice
[Y24-420](https://github.com/sanger/sequencescape/issues/4444) - As LeGo, we would like the new study set up fields to be added to the MLWH
[Y24-421](https://github.com/sanger/sequencescape/issues/4445) - As LeGO, we would like to make sure DAAs which are already created are updated in eDAM, when a Study in Sequencescape is updated
[Y24-422](https://github.com/sanger/sequencescape/issues/4446) - As an SSR, I would like to bulk upload study data from a CSV file generated from the SSR's Study Setup Google Form (Non-MVP)
[Y24-425](https://github.com/sanger/sequencescape/issues/4452) - As LeGo, we want to access the correct DAA in SS, when we get a request for data from an External Researcher

**References**
This story has a non-blocking relationship with:
- Research story [Y24-374](https://github.com/sanger/sequencescape/issues/4407)

**Additional context**

**Comment by harrietc52** [comment]
study first associated with recieving samples
study assocaited with sample data which has been released
ideeally would like first studys access data etc to be shared with the lower groups coming
samples

**Comment by harrietc52** [comment]
Questions

- How would you like to go about testing this? 
- Is testing with each story enough, or would you like some integration / feature test, for example following the above steps?
- Who would be responsible for each step?

**Comment by harrietc52** [comment]
There is an eDAM development environement, which we would need to put in a story to eDAM for it to test pulling in data from MLWH dev

**Comment by harrietc52** [comment]
Duplicate of https://github.com/sanger/sequencescape/issues/4451
